rust/config/mainnet_config.json

```diff
@@ -220,6 +220,51 @@
       "storageGasOracle": "0xBF12ef4B9f307463D3FB59c3604F294dDCe287E2",
       "validatorAnnounce": "0x182E8d7c5F1B06201b102123FC7dF0EaeB445a7B"
     },
+    "blast": {
+      "blockExplorers": [
```

Comment:
I think this is the api endpoint https://api.blastscan.io/api

Reply:
blastscan: https://docs.blastscan.io/getting-started/endpoint-urls
blastexplorer: https://blastexplorer.io/documentation/etherscan-compatibility/accounts

think we've only got the blastexplorer one defined in metadata rn

Reply:
ah good call, thanks